network/policy/policy.go

```diff
@@ -54,6 +63,59 @@ func GetOutBoundNatExceptionList(policies []Policy) ([]string, error) {
 		}
 	}
 
-	log.Printf("OutBoundNAT policy not set.")
+	log.Printf("OutBoundNAT policy not set")
 	return nil, nil
 }
+
+// IsPolicyTypeOutBoundNAT return true if the policy type is OutBoundNAT
+func IsPolicyTypeOutBoundNAT(policy Policy) bool {
+	if policy.Type == EndpointPolicy {
+		type KVPair struct {
```

Comment:
This KVPair is already defined in:
https://github.com/Azure/azure-container-networking/blob/6e6260afe5e3cc0d50969a7e98455e79e1a35148/cni/netconfig.go#L20